core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -212,6 +212,7 @@ public ServerMain(String[] args) throws Exception {
 
             System.gc();
 
+            log.critical().$(buildInformation).$();
```

Comment:
debris ?

Reply:
Now that it is pointed out, yes. I thought I'd send a ping to the alert manager at this point in the start up process, as confirmation that the server is running. But looks hacky, will remove.